sample/ios/Podfile

```diff
@@ -8,8 +8,9 @@ target 'iosApp' do
   use_frameworks!
   pod 'SwiftLint'
   pod 'ViewModelsSample', :path => '../common'
-  pod 'Trikot.viewmodels', :git => 'git@github.com:mirego/trikot.viewmodels.git', :inhibit_warnings => true
   pod 'Trikot.streams', :git => 'git@github.com:mirego/trikot.streams.git', :inhibit_warnings => true
+  pod 'Trikot.viewmodels', :path => '../..', :inhibit_warnings => true
+  pod 'Trikot.viewmodels/Kingfisher', :path => '../..', :inhibit_warnings => true
```

Comment:
Here, should we update the sample app to point to the git repo instead of the local path once a new version of our Pod is deployed?

Reply:
Would make sens IMO, to be able to more easily test locally